src/fmu/dataio/providers/_fmu.py

```diff
@@ -221,11 +221,15 @@ def _validate_and_establish_casepath(self) -> Path | None:
                 return self._runpath.parent
 
         if self.fmu_context == FMUContext.case:
-            # TODO: Change to ValueError when no longer kwargs are accepted in export()
-            warn("Could not auto detect the casepath, please provide it as input.")
+            # TODO: add ValueError when no longer kwargs are accepted in export()
+            ...
 
         logger.debug("No case metadata, issue a warning!")
-        warn("Case metadata does not exist, metadata will be empty!", UserWarning)
+        warn(
```

Comment:
It was a bit many warnings emitted here when the case metadata was not present, so I merged the messages to one